packages/react-components/src/FilterInput.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright 2017-2021 @polkadot/app-accounts authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+import styled from 'styled-components';
+
+import { Input } from '@polkadot/react-components/index';
+
+interface Props {
+  className?: string;
+  filterOn: string;
+  label: string;
+  setFilter: (filter: string) => void;
+}
+
+function Filter ({ className = '', filterOn, label, setFilter }: Props) {
+  return (
+    <div className={`${className} filter--tags`}>
```

Comment:
```suggestion
    <div className={className}>
```
do we use `filter--tags` anywhere?